pkg/recipes/driver/gitconfig.go

```diff
@@ -60,26 +63,36 @@ func getURLConfigKeyValue(secrets v20231001preview.SecretStoresClientListSecrets
 
 	// git config key will be in the format of url.<git url with secret details>.insteadOf
 	// and value returned will the original git url domain, e.g github.com
-	return fmt.Sprintf("url.%s.insteadOf", path), url.Hostname(), nil
+	return fmt.Sprintf("url.%s.insteadOf", path), fmt.Sprintf("%s://%s", url.Scheme, url.Hostname()), nil
 }
 
-// Updates the global Git configuration with credentials for a recipe template path and prefixes the path with environment, application, and resource name to make the entry unique to each recipe execution operation.
+// Updates the local Git configuration in terraform working directory with credentials for a recipe template path and prefixes the path with environment, application, and resource name to make the entry unique to each recipe execution operation.
```

Comment:
`and prefixes the path with environment, application, and resource name to make the entry unique to each recipe execution operation`

Is this still valid?

Reply:
updated the comment